LeaderboardBackend/Controllers/Requests/PasswordAttribute.cs

```diff
@@ -0,0 +1,65 @@
+using System.ComponentModel.DataAnnotations;
+using System.Text.RegularExpressions;
+
+namespace LeaderboardBackend.Controllers.Requests
+{
+	public class PasswordAttribute : ValidationAttribute
+	{
+		private static readonly int MIN = 8;
+		private static readonly int MAX = 80;
+
+		public string GetErrorMessageLength(bool tooShort)
+		{
+			if (tooShort) {
+				return $"Your password needs to be at least {MIN} characters long.";
+			}
+			return $"Your password needs to be at most {MAX} characters long.";
+		}
+
+		public string GetErrorMessageMissing(List<string> missing)
```

Comment:
I think it would be a lot cleaner to just come up with one sentence where it doesn't matter how many errors follow. Maybe something like 
`Your password has the following errors: <all the errors>`